content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md

```diff
@@ -117,43 +117,41 @@ If **Load Balancer** is selected, the following must be true:<p></p><br />
 
 # Deploying a Palette Virtual Cluster
 
-1. From the slide menu, select the **Clusters** tab and click the **Palette Virtual Clusters** tab to list the available Palette Virtual Clusters, and then select **Add New Palette Virtual Cluster**.
+1. From the **Main** menu, click **Clusters** and select a project from the drop-down menu. 
 
+2. Click the **Virtual Clusters** tab to list the available Palette Virtual Clusters, and then select **Add New Virtual Cluster**.
 
-2. Complete the **Deploy New Palette Virtual Cluster** information:<p></p><br />
+3. Complete the **Deploy New Palette Virtual Cluster** information:<br />
```

Comment:
Complete the **Deploy New Virtual Cluster**...
<img width="780" alt="deploy" src="https://user-images.githubusercontent.com/22900687/205198347-d097db61-57fb-42f0-9d34-ad8994284092.png">

Reply:
I thought these labels stay the same because we discussed when the Palette logo doesn't display at top left to give context, the UI will keep the word 'Palette' in labels. Did I misunderstand?

Reply:
@amarjeet27 I see the label now that you point out and removed 'Palette' in the label name.